src/core/webidl-index.js

```diff
@@ -63,5 +63,8 @@ export function run() {
     });
   // Remove duplicate IDs
   pre.querySelectorAll("*[id]").forEach(elem => elem.removeAttribute("id"));
+  pre
```

Comment:
Can we instead copy everything else other than `.idlHeader` instead of removing them?